packages/data/src/plugins/persistence/index.js

```diff
@@ -359,7 +359,10 @@ persistencePlugin.__unstableMigrate = ( pluginOptions ) => {
 		persistence,
```

Comment:
A few lines above this we have a comment about WordPress 6.0. What's up with that? I imagine this stuff needs to stay around forever, no?

Reply:
Hmm, I don't know what the policy is, that note is definitely outdated though. There were some previous very old migrations in the past that had a note to remove them, so I did that. Was that a bad idea? 😄 

Maybe it's worth being a bit more careful now that there are a few preferences that affect accessibility.

Previously the functionality tied to this was fairly minor (fullscreen mode, top toolbar).

Reply:
It depends what the migrations did 😀

If e.g. WordPress 5.0 stored something in `localStorage` in an old format then WordPress 6.0 needs to be able to migrate that.

If it was only ever in the Gutenberg plugin then we can safely delete the migration after a few versions.